manifests/charts/gateway/templates/service.yaml

```
@@ -10,7 +10,7 @@ metadata:
     topology.istio.io/network: "{{.}}"
     {{- end }}
   annotations:
-    {{- .Values.annotations | toYaml | nindent 4 }}
+    {{- merge .Values.service.annotations .Values.annotations | toYaml | nindent 4 }}
```

```suggestion
    {{- merge (deepCopy .Values.service.annotations) .Values.annotations | toYaml | nindent 4 }}
```

Per http://masterminds.github.io/sprig/dicts.html